vcr/issuer/leiaStore.go

```diff
@@ -0,0 +1,82 @@
+package issuer
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	ssi "github.com/nuts-foundation/go-did"
+	"github.com/nuts-foundation/go-did/did"
+	"github.com/nuts-foundation/go-did/vc"
+	"github.com/nuts-foundation/go-leia/v2"
+	"github.com/nuts-foundation/nuts-node/vcr/credential"
+	"path"
+)
+
+// leiaStore implements the issuer Store interface. It is a simple and fast json store.
+// Note: It can not be used in a clustered setup.
+type leiaStore struct {
+	collection leia.Collection
+}
+
+// NewLeiaStore creates a new instance of leiaStore which implements the Store interface.
+func NewLeiaStore(dataDir string) (Store, error) {
```

Comment:
more flexible to pass `dbPath`, since now it makes assumptions about the directory and filename ("vcr" and "issued-credentials").

Maybe even pass the leia instance, might make future merging/splitting of stores easier.